cmd/krel/cmd/announce.go

```diff
@@ -0,0 +1,178 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"k8s.io/release/pkg/http"
+	"k8s.io/release/pkg/mail"
+	"k8s.io/release/pkg/release"
+	"k8s.io/release/pkg/util"
+)
+
+const (
+	sendgridAPIKeyFlag   = "sendgrid-api-key"
+	sendgridAPIKeyEnvKey = "SENDGRID_API_KEY"
```

Comment:
this is for free? or we will have an API key to use

Reply:
Yes it is free but we have an API key to use here

Reply:
i forgot that i saw that :(